### ISSUE TITLE
Invisible field doesn't get overridden field value

### ISSUE THREAD

**Issue by WHITE-developer** [solspace/craft3-freeform#1570]
### What happened?

When using
```
        {% set form = freeform.form("formHandle", {
            fields: {
                invisibleFieldHandle: {
                    value: entry.title,
                },
            },
        }) %}
```
to set a value to an invisible field, it doesn't get saved.
Hidden field does work.

### How can we reproduce this?

1. Add an invisible field type to a form
2. Use the above code
3. Submit the form
4. No override is saved

### Freeform Edition

Pro

### Freeform Version

5.6.5

### Craft Version

4.12.6.1

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

4.1.13

**Comment by gustavs-gutmanis** [comment]
Thanks for reporting this, this has been fixed and will be released in the upcoming version.

**Comment by kjmartens** [comment]
This is resolved in Freeform 5.6.6+ now. 🙂
